frontend/src/test/scala/bloop/TestSpec.scala

```diff
@@ -392,3 +392,25 @@ object JvmTestSpec extends BaseTestSpec("test-project-test", "cross-test-build-s
     }
   }
 }
+
+object NoTestFrameworksSpec extends ProjectBaseSuite("no-test-frameworks") {
```

Comment:
Nice test suite! It was educational for me to see how to test interactions like these